libraries/common/components/I18n/components/Translate/index.jsx

```diff
@@ -13,28 +13,26 @@ import { logger } from '@shopgate/pwa-core';
  * @param {Array} props.children Children to use as placeholders. Must be one of the components
  *                               provided by I18n.
  * @param {string} props.className Additional classes to append to the translated wrapper element.
- * @param {Object} context The component context.
  * @returns {JSX} The translated string as JSX component.
  */
 const Translate = ({
   string,
   children,
   params,
   className,
-}, context) => {
+}) => {
   if (typeof string !== 'string' || string.length === 0) {
     return string;
   }
 
-  if (!context.i18n) {
+  if (!i18n.ready) {
```

Comment:
This should be put into an effect, because otherwise it won't rerender when `i18n` becomes ready.